src/lib.rs

```diff
@@ -955,6 +955,7 @@ impl LightningNode {
                         .to_amount_down(&exchange_rate),
                     breez_payment
                         .amount_msat
+                        .add(breez_payment.fee_msat)
```

Comment:
I agree. As far as I can tell, Breez-SDK doesn't store the invoice and we [only store it in encoded form](https://github.com/getlipa/lipa-lightning-lib/blob/be84175a32329c69dfacad4c502f57e78d78b57f/src/data_store.rs#L36C2-L36C2). 

So in order to fix this, we would first need to persist the invoices properly in the `LocalPaymentData`. Would you agree or am I missing something?

Reply:
Since I'm off for the next couple of days, I'd propose that either you overtake this issue, or that we merge it as is, opening a new issue for the more accurate derivation of the invoiced amount. At least like this, the library returns the correct amount for the time being.